src/main/java/com/faforever/client/login/LoginController.java

```diff
@@ -61,7 +61,14 @@ public void display() {
 
     LoginPrefs loginPrefs = preferencesService.getPreferences().getLogin();
     String username = loginPrefs.getUsername();
-    String password = loginPrefs.getPassword();
+    String password = "";
+    try {
+      password = loginPrefs.getPassword();
+    } catch (Exception e) {
+      loginPrefs.setPassword("");
+      loginPrefs.setAutoLogin(false);
+      logger.warn("Error during password decryption, password was thrown away:", e);
+    }
```

Comment:
We should rather put this into `getPassword` where the *deobfuscation* (not *decryption*) happens, so that  all the hacks stay at the same place and it's much easier to remove later.